server/api/crud/model_monitoring/deployment.py

```diff
@@ -872,10 +872,16 @@ def _delete_model_monitoring_stream_resources(
         )
         stream_paths = []
         for function_name in function_names:
-            label_selector = f"{mlrun_constants.MLRunInternalLabels.nuclio_function_name}={project}-{function_name}"
-            if len(label_selector) > 63:
-                # k8s label character limit exceeded, skipping deletion of stream resources"
+            function_label_name = f"{project}-{function_name}"
```

Comment:
Is this the label name or rather the label value? Or maybe better name it `qualified_function_name`?